entrypoint/cmd/entry.go

```diff
@@ -59,10 +61,11 @@ Wrapper for COSA commands and templates`,
 	}
 
 	cmdSteps = &cobra.Command{
-		Use:   "run-steps",
-		Short: "Run Steps from [file]",
-		Args:  cobra.MinimumNArgs(1),
-		Run:   runSteps,
+		Use:          "run-steps",
```

Comment:
If we're bike-shedding, it would be `run-scripts`, since it takes N arguments.  However, I'll concede your bike shed on the condition of fixing that in a follow-up.  The naming of "run-steps" betrays some thinking I've been doing on an interface on dependencies. And I'm not entirely sure what I want to propose there.

Reply:
Bikeshed implemented.